python/lsst/daf/base/propertyContainer/propertyContainerContinued.py

```diff
@@ -36,18 +36,21 @@
 from ..dateTime import DateTime
 
 
-def getPropertySetState(container):
+def getPropertySetState(container, asList=False):
```

Comment:
I might slightly prefer `asLists` rather than `asList`, as it's a list of lists or a list of tuples.